test/test-framework/pkg/apis/cache/group.go

```diff
@@ -0,0 +1,15 @@
+// Copyright 2018 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cache
```

Comment:
IIRC, this file is in the `add api` scaffolds to help silence a warning during code generation. I don't think it hurts anything, but I don't think it is necessary either.

Reply:
Oks, I will remove.

Reply:
Done